lti_consumer/outcomes.py

```diff
@@ -0,0 +1,205 @@
+"""
+This module adds support for the LTI Outcomes Management Service.
+
+For more details see:
+https://www.imsglobal.org/specs/ltiomv1p0
+"""
+
+import logging
+import urllib
+
+from lxml import etree
+from xml.sax.saxutils import escape
+
+from xblockutils.resources import ResourceLoader
+
+from .exceptions import LtiError
+from .oauth import verify_oauth_body_signature
+
+
+log = logging.getLogger(__name__)
+
+
+def parse_grade_xml_body(body):
```

Comment:
Would it be safe enough to assume that such attacks would be handled by the web-tier (`client_max_body_size` on nginx)? I will also add a Jira issue for changing this to parse the XML body iteratively as a stream.

https://openedx.atlassian.net/browse/LTI-7